src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/services/servicebus/ServiceBusHelperTest.java

```diff
@@ -61,22 +85,47 @@ public void should_send_message_with_messageId() throws Exception {
 
     @Test(expected = InvalidMessageException.class)
     public void should_throw_exception_for_empty_messageId() {
-        Msg msg = new EnvelopeMsg("");
+        when(envelope.getId()).thenReturn(null);
+        Msg msg = new EnvelopeMsg(envelope);
         serviceBusHelper.sendMessage(msg);
     }
 
     @Test
-    public void should_add_test_label_to_test_message() {
-        Msg msg = new EnvelopeMsg("envelopeId", true);
+    public void should_not_add_any_label_to_standard_message() {
+        Msg msg = new EnvelopeMsg(envelope);
         Message busMessage = serviceBusHelper.mapToBusMessage(msg);
-        assertThat(busMessage.getLabel()).isEqualTo(MsgLabel.TEST.toString());
+        assertThat(busMessage.getLabel()).isNullOrEmpty();
     }
 
     @Test
-    public void should_not_add_any_label_to_standard_message() {
-        Msg msg = new EnvelopeMsg("envelopeId");
-        Message busMessage = serviceBusHelper.mapToBusMessage(msg);
-        assertThat(busMessage.getLabel()).isNullOrEmpty();
+    public void should_send_message_with_envelope_data() throws Exception {
+
+        Msg message = new EnvelopeMsg(envelope);
+        Message busMessage = serviceBusHelper.mapToBusMessage(message);
+
+        JsonNode jsonNode = objectMapper.readTree(busMessage.getBody());
+
+        assertThat(jsonNode.get("case_ref").textValue()).isEqualTo("1111222233334446");
+        assertThat(jsonNode.get("jurisdiction").textValue()).isEqualTo("SSCS");
+        assertThat(jsonNode.get("zipFileName").textValue()).isEqualTo("zip-file-test.zip");
```

Comment:
perhaps we could use the values retrieved from `envelope`? Otherwise it's hard to tell, why exactly `1111222233334446` etc is the correct value.

Reply:
Sure